packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx

```diff
@@ -95,10 +95,12 @@ const GetExpandedColumns = (
           tooltip: "העתק",
           iconUrl: copyIcon,
           onClick: () => {
-            let bookingDate = appointment.donationStartTimeMillis
-              ? DateUtils.ToDateString(appointment.donationStartTimeMillis)
+            const bookingDate = appointment.donationStartTimeMillis
+              ? DateUtils.ToDateString(appointment.donationStartTimeMillis) +
+                " " +
+                DateUtils.ToTimeString(appointment.donationStartTimeMillis)
```

Comment:
Why is the start time here twice?

Also, please extract this `appointment -> copy string` to a function in an external file and add tests to it